tests/selenium/spec/code-layout-spec.js

```diff
@@ -11,7 +11,8 @@ describe('code page spec (java/spring/)', () => {
   const codePage = new CodePage('/code/java/spring/');
 
   beforeEach(util.itHelper(async () => {
-    await codePage.navigate();
+    await codePage.navigate('/code/java/spring/');
+    await codePage.refresh();
```

Comment:
Can you explain the need for a refresh before each test?

Reply:
The refresh feels like a hack, but I noticed that there are a couple of things that seem to work better (more consistently). One is checking the current URL. The other seems to be finding elements or checking active link text. I noticed when I was running these tests that this one was consistently having to retry to pass.